src/ChangeStream.php

```diff
@@ -55,6 +57,8 @@ public function __construct(Cursor $cursor, callable $resumeCallable)
         $this->csIt = new IteratorIterator($cursor);
 
         $this->key = 0;
+
+        $this->previousValid = false;
```

Comment:
Can we initialize `$key` and `$previousValid` in the `private` declarations above? That would reduce a few lines in the constructor.